src/Widgets/EtherInterface.vala

```diff
@@ -19,6 +19,8 @@
 public class Network.EtherInterface : Network.WidgetNMInterface {
     private Wingpanel.Widgets.Switch ethernet_item;
 
+    public string active_wired_name { get; private set; }
```

Comment:
Because this is a subclass of [Network.WidgetNMInterface](https://github.com/elementary/wingpanel-indicator-network/blob/master/src/common/Widgets/WidgetNMInterface.vala), `display_title` is already a public property so you don't need to add another property here